gherkin-utils/java/gherkin-utils/src/main/java/gherkin/utils/walker/model/RejectAllFilters.java

```diff
@@ -0,0 +1,25 @@
+package gherkin.utils.walker.model;
+
+import io.cucumber.messages.Messages;
+
+public abstract class RejectAllFilters extends DefaultFilters {
```

Comment:
This class does not have to be abstract.

Reply:
I think with abstract class, when we want to custom the ```Filter```, we don't need to implement all method of the interface ```Filter```. E.g:
```
GherkinDocumentWalker walker = new GherkinDocumentWalker(new RejectAllFilters() {
            @Override
            public boolean acceptScenario(Messages.GherkinDocument.Feature.Scenario scenario) {
                return scenario.getName().equals("Earth");
            }
        }, new DefaultHandlers() {
        });
```

Reply:
That's because you've made `RejectAllFilters` extend `DefaultFilter` a class with an implementation. If you create `Filter` interface with an `RejectAllFilter` and `AcceptAllFilter` implementation you don't have this problem. And because `AcceptAllFilter` and `RejectAllFilter` are not final, you can still extend them.

Reply:
Thank you @mpkorstanje, I have already changed like your suggestion.